src/components/WorkspaceList/WorkspaceList.tsx

```diff
@@ -84,6 +96,7 @@ const WorkspaceList: React.FC = () => {
           filters={workspaceFilters}
           rowActions={workspaceActions}
           emptyStateDescription="No data was retrieved" // TODO: Add check so that empty payload results in the "Get Started with Workspaces" UI
+          actionButtons={actionButtons}
```

Comment:
FYI to @fhlavac - this is the prop we're now passing into the list view to put the workspace add button. We will need to update this with the FEC toolbar work.